examples/chip-tool/commands/common/Command.h

```diff
@@ -88,6 +89,7 @@ class Command
         return AddArgument(name, min, max, reinterpret_cast<void *>(out));
     }
     size_t AddArgument(const char * name, const char * value);
+    size_t AddArgument(const char * name, char ** value);
```

Comment:
I would maybe add some doc comments here, to understand the difference between const char * and char ** arguments